code/services/FreshdeskSSO.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class FreshdeskSSO extends Controller
```

Comment:
This has no dependency on the userforms component. I think it could be conceivably spinned out?

Reply:
It's part of the functionality the module offers. I don't think we should spin it out.